modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/java/com/liferay/dynamic/data/mapping/util/DDMFormValuesToFieldsConverterTest.java

```diff
@@ -89,22 +90,39 @@ public void testConversionWithBooleanField() throws Exception {
 		Fields fields = _ddmFormValuesToFieldsConverter.convert(
 			ddmStructure, ddmFormValues);
 
-		Assert.assertNotNull(fields);
+		_assertBooleanFieldValue(true, fields);
 
-		Field field = fields.get("Boolean");
+		Field fieldsDisplayField = fields.get(DDMImpl.FIELDS_DISPLAY_NAME);
 
-		Serializable value = field.getValue();
+		Assert.assertEquals(
+			"Boolean_INSTANCE_rztm", fieldsDisplayField.getValue());
 
-		Class<?> clazz = value.getClass();
+		// predefinedValue
```

Comment:
Dear reviewer, 
I'm not very sure about the comment, or if it's appropriate create a new test method, instead of keep my test inside this one. Feel free to give some suggestion